libraries/standard/mqtt/src/iot_mqtt_validate.c

```diff
@@ -148,9 +184,10 @@ bool _IotMqtt_ValidateConnect( const IotMqttConnectInfo_t * pConnectInfo )
 }
 
 /*-----------------------------------------------------------*/
-
-bool _IotMqtt_ValidatePublish( bool awsIotMqttMode,
-                               const IotMqttPublishInfo_t * pPublishInfo )
+static bool _mqtt_ValidatePublish( bool awsIotMqttMode,
```

Comment:
This function does not follow any naming convention.

It should be called _validatePublish.

Also extra newline needed before 187.

Reply:
Renamed to `_validatePublish()`, newline added.